Patch from Andrew Resch at https://bugs.gentoo.org/show_bug.cgi?id=139534
Also present in upstream svn tree

--- libgphoto2-2.2.1/camlibs/sonix/library.c	2006-04-18 18:54:18.000000000 -0700
+++ libgphoto2-2.2.1-work/camlibs/sonix/library.c	2006-07-07 23:12:56.012649500 -0700
@@ -41,7 +41,7 @@
 #else
 #  define _(String) (String)
 #  define N_(String) (String)
-#  define ngettext(String1,String2,Count) ((Count==1)?String1:String)
+#  define ngettext(String1,String2,Count) ((Count==1)?String1:String2)
 #endif
 
 #include <gphoto2-port.h>
